editor/sidebar/post-taxonomies/index.js

```diff
@@ -60,7 +54,7 @@ class PostTaxonomies extends Component {
 		}
 
 		return (
-			<PanelBody title={ __( 'Categories & Tags' ) } opened={ this.props.isOpened } onToggle={ this.onToggle }>
+			<PanelBody title={ __( 'Categories & Tags' ) } opened={ this.props.isOpened } onToggle={ this.props.onTogglePanel }>
```

Comment:
Minor: Long line could do for either splitting onto multiple lines and/or destructuring props:

```jsx
const { isOpened, onTogglePanel } = this.props;

return (
	<PanelBody 
		title={ __( 'Categories & Tags' ) }
		opened={ isOpened }
		onToggle={ onTogglePanel }
	>
```